Dockerfile

```
@@ -0,0 +1,36 @@
+##
+## Build
+##
+FROM golang:1.19 AS build
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+
+# Cache dependencies before building and copying the source code, so that
+# we don't need to re-download when building and so that the change in the
+# source code do not invalidate our downloaded layer.
+RUN go mod download
+
+COPY ./src ./src
+
+RUN go build -o /sfu ./src
```

```suggestion
RUN go build -o /waterfall ./src
```

I think we'd prefer to call the binary `waterfall`

---

Ok, let me change it to `waterfall` in all places. No strong opinion :)